bin/hdfcoinc/pycbc_fit_sngls_by_template

```diff
@@ -289,8 +289,15 @@ counts_above = []
 fits = []
 tpars = []
 
-for tnum in trange:
-    stat_in_template = stat[tid == tnum]
+tsort = tid.argsort()
+tid = tid[tsort]
+stat = stat[tsort]
+
+left = np.searchsorted(tid, trange, side='left')
```

Comment:
1 line of comment here explaining that there will be multiple entries of the same tid in the sorted array and left/right catch the appropriate range? 
Also do we know if it works if/when some templates have zero triggers?

Reply:
It works when there are zero triggers in my test. In that case, left and right end up the same value and if you follow the later logic you get empty arrays for the stat values.

Reply:
This was not such an uncommon case in my use actually given the number of templates and how few triggers I kept, there were some templates without triggers. (thankfully the average fit was very smooth anyway over a large region).